packages/feedback/src/core/integration.ts

```diff
@@ -141,7 +142,8 @@ export const feedbackIntegration = (({
     if (!client) {
       throw new Error('Sentry Client is not initialized correctly');
     }
-    const modalIntegration = client.getIntegrationByName<FeedbackModalIntegration>('FeedbackModal');
```

Comment:
Is it a problem to re-add it if it's already been added? This integration especially has no state, we're using it to pass around some function references

Reply:
Actually, I'm going to leave it because with out [desired interface](https://github.com/getsentry/sentry-javascript/issues/11435#issuecomment-2038271957) the user shouldn't be manually adding things (they never did, never need to). 

This spot is where the call to `Sentry.lazyLoadIntegration()` will happen, so we'll tweak it and include that line as well in a followup PR